### PR TITLE
Changed cluster-etcd-operator single-node aws e2e job for master-presubmit to always_run

### PR DESCRIPTION
**Pull request by omertuc**
Since the single-node aws e2e job has recently stabilized to a point where there's a good chance it succeeds, we're enabling it to always run on all openshift components where it is defined.

This will give us more data to troubleshoot the reasons it is failing and it will also serve as a first step before making this job mandatory for PR merges

### CODE REVIEW

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Comment by hexfusion** [comment]
/hold

I believe if we are going to block it should be on the metal sno test cc @romfreiman

**Comment by osherdp** [comment]
@hexfusion but it's ``optional: true``, no?

**Comment by omertuc** [comment]
> @hexfusion but it's `optional: true`, no?

We're discussing it on Slack

**Comment by hexfusion** [comment]
my argument is why even bother running a test that is not reflective of what we are shipping.... AWS flow has known etcd bugs as you actually have 2 nodes in the process vs 1. So if we are going to run a test it should be metal but I don't feel that strongly 

/hold cancel
/lgtm

**Comment by hexfusion** [comment]
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20382/rehearse-20382-pull-ci-openshift-cluster-etcd-operator-master-e2e-aws-single-node/1417237818903629824
```
Jul 19 23:25:35.667: INFO: operator=etcd-operator, watchrequestcount=386, upperbound=250, ratio=1.544
Jul 19 23:25:35.667: INFO: Operator etcd-operator produces more watch requests than expected
```
looks like we have  hard fail out of the box........................................

**Comment by omertuc** [comment]
> my argument is why even bother running a test that is not reflective of what we are shipping.... AWS flow has known etcd bugs as you actually have 2 nodes in the process vs 1. So if we are going to run a test it should be metal but I don't feel that strongly
> 
> /hold cancel
> /lgtm

It's not perfect but it's the only way for us to run conformance tests on single-node openshift, since conformance tests don't work on metal OpenShift in general (not just on SNO). It's not perfect and it might cause us to chase bugs we'll never ship / miss bugs we might ship, but it also might help us catch bugs that apply to both flows before they're merged.

We are also running a subset of conformance tests on the live-cd BIP flow, just nightly instead of on repo master presubmits.

**Pull request merged**
